src/test/java/uk/gov/digital/ho/hocs/info/api/TeamResourceTest.java

```diff
@@ -44,7 +43,7 @@ public void shouldAddUserToTeam() {
 
         ResponseEntity result = teamResource.addUserToGroup(userUUID.toString(), teamUUID.toString());
         assertThat(result.getStatusCode()).isEqualTo(HttpStatus.OK);
-        verify(teamService, times(1)).addUserToTeam(userUUID, teamUUID);
```

Comment:
This checks that we only call the addUserToTeam once, which has now been removed.
If we are entering it more times due to the code change, then the number should be used instead of just verifying that it is called. This change has lowered the standard of the test.

Reply:
true, will fix that

Reply:
Actually i just realised why I've done it in the first place :D. The verify(xx, times(1)) is redundant as verify(xx) will default to times(1) in the background. So we are safe to remove these when the value of times is 1.